bin/atlassian-backup

```diff
@@ -61,7 +73,14 @@ function backup_attachments() {
 function dump_database() {
     echo "  - Dumping $NATURAL_TOOL_NAME database"
     DATABASE_DUMP_OUTPUT="${OUTPUT_PREFIX}-database-dump.sql.gz"
-    $PGENGINE/pg_dump "$DB_NAME" | gzip > "$DATABASE_DUMP_OUTPUT"
+    if [ "${DB_BACKEND}" = 'mysql' ]; then
+        $MYENGINE --single-transaction --host="$MYSQL_HOST" --user="$MYSQL_USER" --password="$MYSQL_PASS" "$DB_NAME" | gzip > "$DATABASE_DUMP_OUTPUT"
```

Comment:
So sorry I forgot this last time:
$MYENGINE/mysqldump

Reply:
Did you want me to submit this as well?